core-httpclient-impl/README.md

```diff
@@ -139,20 +151,49 @@ The polling interval is used to specify a fixed delay between consecutive HTTP r
 #### Initial datafile
 
 You can provide an initial datafile via the builder to bootstrap the `ProjectConfigManager` so that it can be used
-immediately without blocking execution.
+immediately without blocking execution. This also serves as a "fallback" datafile if no HTTP connection can be established.
+This can be useful in a mobile environment where internet connectivity is not always guaranteed. This datafile will be
+discarded, however, after the first successful datafile poll.
+
+### Builder Methods
+The following builder methods can be used to custom configure the `HttpProjectConfigManager`.
+
+|Builder Method|Default Value|Description|
+|---|---|---|
+|`withDatafile(String)`|null|Initial datafile, typically sourced from a local cached source.|
+|`withUrl(String)`|null|Url override location used to specify custom HTTP source for the Optimizely datafile.|
+|`withFormat(String)`|https://cdn.optimizely.com/datafiles/%s.json|Parameterized datafile URL by SDK key.|
+|`withPollingInterval(Long, TimeUnit)`|5 minutes|Fixed delay between fetches for the datafile.|
```

Comment:
Does the value actually include "minutes"? Or is it more like the default value is 5 and the description should say "In minutes."

Reply:
The method takes two values, the "period" and the "time unit" so I think saying "5 minutes" is appropriate.